pkgs/servers/http/apache-httpd/2.2.nix

```diff
@@ -23,6 +23,7 @@ stdenv.mkDerivation rec {
   # FIXME: -dev depends on -doc
   outputs = [ "dev" "out" "doc" ];
   setOutputFlags = false; # it would move $out/modules, etc.
+  stripDebugList= [ "bin" "lib" "modules" ]; # also strip modules.
```

Comment:
What is the purpose of this modification? This doesn't seem to change anything, really. Am I missing something?